docs/contributing.md

```diff
@@ -150,3 +150,12 @@ and then pull the image and retag it to the name expacted by `syz-env`:
 docker pull docker.pkg.github.com/google/syzkaller/env
 docker tag docker.pkg.github.com/google/syzkaller/env gcr.io/syzkaller/env
 ```
+
+### Using [act](https://github.com/nektos/act)
+.github/workflows has more tests compared to `syz-env make presubmit`. To have the same tests as the workflow, we can run these workflow jobs locally.
+```
+curl https://raw.githubusercontent.com/nektos/act/master/install.sh | sudo bash
```

Comment:
Let's recommend to follow their documentation at https://github.com/nektos/act?tab=readme-ov-file#manually-building-from-source .
"curl .../install.sh | sudo bash" looks dangerous as a general recommendation.